docs/Controllers.md

```diff
@@ -0,0 +1,85 @@
+# Controllers
+
+_(c) AMWA 2022, CC Attribution-NoDerivatives 4.0 International (CC BY-ND 4.0)_
+
+## Informative
+
+A Controller is Client software that interacts with the NMOS APIs to discover, connect and manage devices within a networked media system.
+
+* This document includes normative references to be followed when implementing a Controller.
+* This document covers how the Controller interacts with the NMOS APIs only.
+It does not cover other features of the Controller software, such as presentation.
+* This document does not cover any requirements relating to where a Controller is additionally acting as a Node (e.g. receiving monitoring information via IS-07).
+* Where this document refers to the "user" of a Controller, this includes both human operators who drive the Controller manually and automation systems that drive the Controller programmatically.
+
+## General
+
+### HTTP APIs
+#### Trailing Slashes
+APIs may advertise URLs with or without a trailing slash.
+Controllers appending paths to `href` type attributes MUST support both cases, avoiding doubled or missing slashes.
+
+Controllers performing requests other than GET or HEAD (i.e PUT, POST, DELETE, OPTIONS etc.) MUST use URLs with no trailing slash present.
+
+#### API Version
+Controllers are responsible for identifying the correct API version they require.
+
+Implementers of Controllers are strongly recommended to support multiple versions of the NMOS APIs simultaneously in order to ease the upgrade process in live facilities.
+
+Later API versions might use URNs which have not yet been defined and so Controllers MUST be tolerant to these.
+
+#### Error Codes & Responses
+The NMOS APIs use HTTP status codes to indicate success, failure and other cases to Controllers as per [RFC 7231](https://tools.ietf.org/html/rfc7231) and related standards.
+Where the RAML specification of an API specifies explicit response codes it is expected that a Controller will handle these cases in a particular way.
```

Comment:
Maybe expected can be replaced with normative language? 🤔 The next sentence seems to oppose this a little

Reply:
Yes, good point. For reference, this section was lifted verbatim from [IS-08](https://specs.amwa.tv/is-08/releases/v1.0.1/docs/2.0._APIs.html#error-codes--responses) so same probably applies there.